Examples/Blink/Core/Src/main.cpp

```diff
@@ -87,8 +90,19 @@ int main(void)
   /* Initialize all configured peripherals */
   MX_GPIO_Init();
   MX_USART2_UART_Init();
+  MX_SPI2_Init();
   /* USER CODE BEGIN 2 */
 
+  // SPIInterface test for normal transmit and recieve functions
+  SPIInterface spi2(&hspi2);
+
+  uint8_t message[13] = "Hello World!";
+  spi2.Transmit(message, 13, 100);
+
+  // This is just placeholder as I don't know what it'll be tested with
+  spi2.Receive(20, 100);
+  spi2.Transmit(spi2.getReceivedData(), 20, 100);
+

```

Comment:
Yeah this is good for now, once we get a few of these merged we will start writing some test documentation and getting formal test cases set up